README.md

```diff
@@ -53,6 +53,13 @@ Any static assets, like images, can be placed in the `public/` directory.
 
 ## Deployment
 
+### Setting up Environment Variables
+
+1. Run `npx partykit@latest token generate` to generate a token.
+2. Run `ntl env:set PARTYKIT_LOGIN choose-your-value-here` to add it to the environment variables for Netlify deploys.
```

Comment:
I wasn't sure what to use as a placeholder here for the value.

The way I understood the [docs](https://cli.netlify.com/commands/env#envset), was that `env:set` command accepts a key value pair.
In the open issue, the placeholder value was set to `nickytonline` but I am assuming people using this for their own personal projects will want to use their own username.

So maybe this should say `ntl env:set PARTYKIT_LOGIN add-username-here`?

Reply:
> So maybe this should say ntl env:set PARTYKIT_LOGIN add-username-here?

I'd go with `ntl env:set PARTYKIT_LOGIN your-username`

Reply:
The template is meant to deploy to Netlify. They can obviously deploy elsewhere, so I wonder if we should explicitly mention that `ntl env:set` is for Netlify only and if you deploy elsewhere to refer to their docs on setting environment variables for an application?

Reply:
> so I wonder if we should explicitly mention that ntl env:set is for Netlify only and if you deploy elsewhere to refer to their docs on setting environment variables for an application?

Yeah, I think it is fine to add a note like that